independent-projects/tools/common/src/main/java/io/quarkus/devtools/project/extensions/ExtensionsManager.java

```diff
@@ -0,0 +1,109 @@
+package io.quarkus.devtools.project.extensions;
+
+import io.quarkus.bootstrap.model.AppArtifactCoords;
+import io.quarkus.bootstrap.model.AppArtifactKey;
+import io.quarkus.dependencies.Extension;
+import io.quarkus.devtools.project.BuildTool;
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ * This interface defines a high level way of managing (read/write) extensions in any QuarkusProject
+ */
+public interface ExtensionsManager {
```

Comment:
Better use the singular form here: `ExtensionManager`. eg. https://github.com/quarkusio/quarkus-registry-descriptor/blob/master/src/main/java/io/quarkus/registry/ExtensionRegistry.java

Reply:
ok I'll change